File: PST/FUEL/src/FUEL/clsUploadSettings2.vb
Imports System
Imports System.Diagnostics
Imports System.IO
Imports System.Runtime.CompilerServices

Namespace FUEL
    Public Class clsUploadSettings2
        ' Methods
        Public Sub Reset()
            Me._InstallLocation = TestSites.HP
            Me._LocalCopyToAddress = "M:\"
            Me._VCDServerAddress = "http://pst1.vcd.hp.com/PST/upload.php"
            Me._SettingsVerified = False
            Me._DebugLevel = 4
        End Sub


        ' Properties
        Private Property _InstallLocation As TestSites
            <DebuggerNonUserCode> _
            Get
                Return Me.__InstallLocation
            End Get
            <DebuggerNonUserCode> _
            Set(ByVal AutoPropertyValue As TestSites)
                Me.__InstallLocation = AutoPropertyValue
            End Set
        End Property

        Private Property _VCDServerAddress As String
            <DebuggerNonUserCode> _
            Get
                Return Me.__VCDServerAddress
            End Get
            <DebuggerNonUserCode> _
            Set(ByVal AutoPropertyValue As String)
                Me.__VCDServerAddress = AutoPropertyValue
            End Set
        End Property

        Private Property _LocalCopyToAddress As String
            <DebuggerNonUserCode> _
            Get
                Return Me.__LocalCopyToAddress
            End Get
            <DebuggerNonUserCode> _
            Set(ByVal AutoPropertyValue As String)
                Me.__LocalCopyToAddress = AutoPropertyValue
            End Set
        End Property

        Private Property _SettingsVerified As Boolean
            <DebuggerNonUserCode> _
            Get
                Return Me.__SettingsVerified
            End Get
            <DebuggerNonUserCode> _
            Set(ByVal AutoPropertyValue As Boolean)
                Me.__SettingsVerified = AutoPropertyValue
            End Set
        End Property

        Private Property _DebugLevel As Integer
            <DebuggerNonUserCode> _
            Get
                Return Me.__DebugLevel
            End Get
            <DebuggerNonUserCode> _
            Set(ByVal AutoPropertyValue As Integer)
                Me.__DebugLevel = AutoPropertyValue
            End Set
        End Property

        Friend ReadOnly Property SettingsFileLocation As String
            Get
                Return If(Not modCommonCode.GetDataPath.ToLower.Contains("\bin\"), Path.Combine(modCommonCode.GetDataPath, "Uploadsettings2.xml"), "C:\Users\morrisor\Documents\Visual Studio 2010\Projects\FUEL\AutoSendFiles\FUEL\Uploadsettings2.xml")
            End Get
        End Property

        Public Property InstallLocation As TestSites
            Get
                Return Me._InstallLocation
            End Get
            Set(ByVal value As TestSites)
                Me._InstallLocation = value
            End Set
        End Property

        Public Property VCDServerAddress As String
            Get
                Return Me._VCDServerAddress
            End Get
            Set(ByVal value As String)
                Me._VCDServerAddress = value
            End Set
        End Property

        Public Property LocalCopyToAddress As String
            Get
                Return Me._LocalCopyToAddress
            End Get
            Set(ByVal value As String)
                Me._LocalCopyToAddress = value
            End Set
        End Property

        Public Property SettingsVerified As Boolean
            Get
                Return Me._SettingsVerified
            End Get
            Set(ByVal value As Boolean)
                Me._SettingsVerified = value
            End Set
        End Property

        Public Property DebugLevel As Integer
            Get
                Return Me._DebugLevel
            End Get
            Set(ByVal value As Integer)
                Me._DebugLevel = value
            End Set
        End Property


        ' Fields
        <CompilerGenerated, DebuggerBrowsable(DebuggerBrowsableState.Never)> _
        Private __InstallLocation As TestSites
        <CompilerGenerated, DebuggerBrowsable(DebuggerBrowsableState.Never)> _
        Private __VCDServerAddress As String
        <DebuggerBrowsable(DebuggerBrowsableState.Never), CompilerGenerated> _
        Private __LocalCopyToAddress As String
        <DebuggerBrowsable(DebuggerBrowsableState.Never), CompilerGenerated> _
        Private __SettingsVerified As Boolean
        <CompilerGenerated, DebuggerBrowsable(DebuggerBrowsableState.Never)> _
        Private __DebugLevel As Integer
    End Class
End Namespace

File: PST/FUEL Source Code/FUEL/Backup/clsUploadSettings2.cs
// Decompiled with JetBrains decompiler
// Type: FUEL.clsUploadSettings2
// Assembly: FUEL, Version=2.4.8.0, Culture=neutral, PublicKeyToken=null
// MVID: A3D630D0-0620-425D-8CFB-B67B2AC75507
// Assembly location: C:\hp\FUEL\FUEL.dll

using System.Diagnostics;
using System.IO;

namespace FUEL
{
  public class clsUploadSettings2
  {
    private PST.TestSites _InstallLocation { [DebuggerNonUserCode] get; [DebuggerNonUserCode] set; }

    private string _VCDServerAddress { [DebuggerNonUserCode] get; [DebuggerNonUserCode] set; }

    private string _LocalCopyToAddress { [DebuggerNonUserCode] get; [DebuggerNonUserCode] set; }

    private bool _SettingsVerified { [DebuggerNonUserCode] get; [DebuggerNonUserCode] set; }

    private int _DebugLevel { [DebuggerNonUserCode] get; [DebuggerNonUserCode] set; }

    internal string SettingsFileLocation => modCommonCode.GetDataPath().ToLower().Contains("\\bin\\") ? "C:\\Users\\morrisor\\Documents\\Visual Studio 2010\\Projects\\FUEL\\AutoSendFiles\\FUEL\\Uploadsettings2.xml" : Path.Combine(modCommonCode.GetDataPath(), "Uploadsettings2.xml");

    public PST.TestSites InstallLocation
    {
      get => this._InstallLocation;
      set => this._InstallLocation = value;
    }

    public string VCDServerAddress
    {
      get => this._VCDServerAddress;
      set => this._VCDServerAddress = value;
    }

    public string LocalCopyToAddress
    {
      get => this._LocalCopyToAddress;
      set => this._LocalCopyToAddress = value;
    }

    public bool SettingsVerified
    {
      get => this._SettingsVerified;
      set => this._SettingsVerified = value;
    }

    public int DebugLevel
    {
      get => this._DebugLevel;
      set => this._DebugLevel = value;
    }

    public void Reset()
    {
      this._InstallLocation = PST.TestSites.HP;
      this._LocalCopyToAddress = "M:\\";
      this._VCDServerAddress = "http://pst1.vcd.hp.com/PST/upload.php";
      this._SettingsVerified = false;
      this._DebugLevel = 4;
    }
  }
}
